taskcluster/ci/config.yml

```diff
@@ -37,6 +37,11 @@ workers:
             implementation: docker-worker
             os: linux
             worker-type: b-linux
+        b-linux-large:
+            provisioner: 'mozillavpn-{level}'
+            implementation: docker-worker
+            os: linux
+            worker-type: b-linux-large
```

Comment:
Oh, I think I gave you some bad information. Looks like this might need to be `b-linux-large-gcp`.. Sorry!

You can keep the name of this config as `b-linux-large` and that way you can still use `b-linux-large` in the task definitions.